dll/ntdll/rtl/libsupp.c

```diff
@@ -819,6 +819,72 @@ RtlDosApplyFileIsolationRedirection_Ustr(IN ULONG Flags,
         return STATUS_SXS_KEY_NOT_FOUND;
     }
 
+    if (NtCurrentPeb()->ProcessParameters &&
+        (NtCurrentPeb()->ProcessParameters->Flags & RTL_USER_PROCESS_PARAMETERS_PRIVATE_DLL_PATH))
+    {
+        UNICODE_STRING RealName, LocalName;
+        WCHAR RealNameBuf[MAX_PATH], LocalNameBuf[MAX_PATH];
+
+        RtlInitEmptyUnicodeString(&RealName, RealNameBuf, sizeof(RealNameBuf));
+        RtlInitEmptyUnicodeString(&LocalName, LocalNameBuf, sizeof(LocalNameBuf));
+
+        Status = RtlComputePrivatizedDllName_U(OriginalName, &RealName, &LocalName);
+        if (!NT_SUCCESS(Status))
+        {
+            DPRINT1("RtlComputePrivatizedDllName_U failed: 0x%lx\n", Status);
+            return Status;
+        }
+
+        if (RtlDoesFileExists_UStr(&LocalName))
+        {
+            Status = get_buffer(&fullname, LocalName.Length + sizeof(WCHAR), StaticString, DynamicString != NULL);
```

Comment:
`sizeof(UNICODE_NULL)` here and 3x below

Reply:
I'll add an assert that this is the same size, and replace em all.